### ISSUE TITLE
Boot time: add a section about reducing Boot time on a system built by Yocto

### ISSUE THREAD

**Issue by michaelopdenacker** [bootlin/training-materials#75]
Ideally, we should have two sections: one for systems built by Buildroot, one for systems built by Yocto.

**Comment by tpetazzoni** [comment]
Does it really make sense to tie this to build systems? To me, boot time optimization is largerly build system independent.

**Comment by michaelopdenacker** [comment]
Right, but it's probably helpful to have a quick reference about how to implement the various optimizations on each build system.
